aivalidation_dictionary.txt

```diff
@@ -32,6 +33,8 @@ interpretability
 ITerm
 Kubeflow
 Laurens
+LLM
+LLM's
```

Comment:
The plural of LLMs seems to be LLMs: https://en.wiktionary.org/wiki/LLMs#English